homework/12_A/18_Lachezar_Lazarov/homework01/test_homework1.py

```diff
@@ -0,0 +1,28 @@
+from math import sqrt
+
+def checkCircle(c1_center, c1_rad, c2_center, c2_rad):
+    if (c1_center == c2_center and c1_rad == c2_rad):
+	    return "Matching"
```

Comment:
Using hardcoded string decreases changeability. It is better if you'd used an Enum.